Content/NPCs/Town/BarkeepNPC.cs

```diff
@@ -122,11 +124,6 @@ public override void OnChatButtonClicked(bool firstButton, ref string shopName)
 		{
 			shopName = "Shop";
 		}
-		/*else
-		{
-			Main.npcChatText = Language.GetTextValue("Mods.PathOfTerraria.NPCs.Blacksmith.Dialogue.Quest");
-			Main.LocalPlayer.GetModPlayer<QuestModPlayer>().StartQuest<BlacksmithStartQuest>();
-		}*/
```

Comment:
Intentional comments?

Reply:
I don't know how, but this comment and the one on the EoW portal simply don't exist in my code. Not an issue I guess?